views/halloffame.html

```diff
@@ -69,6 +83,81 @@
 			</section >
 			<section class="hof-stats right-side">
 				<h2>My Stats</h2>
+				<div class="stats-helper">
+					<p><span class="regNum"></span> regular HOF conversations</p>
+					<p><span class="gpNum"></span> gold plated conversations</p>
+					<p><span class="gNum"></span> golden conversations</p>
+				</div>
+				<div id="chart"></div>
+			    <script>
+				    var $window = $(window);
+		
+			      	(function(d3) {
+			        'use strict';
```

Comment:
Good point. I took an example code snippet I found only and modified it to fit here. So yes, we don't need `use strict` nor do we need the function wrapping.